src/privileged/blurts/subscripts/PanelUI.jsm

```diff
@@ -0,0 +1,138 @@
+class PanelUI {
+  constructor(doc) {
+    this.site = null;
+    this.doc = doc;
+    const box = doc.createElementNS(XUL_NS, "vbox");
+    box.className = "container";
+
+    let elt, elt2, elt3;
+
+    elt = doc.createElementNS(XUL_NS, "description");
+    elt.appendChild(doc.createTextNode("Have an account? It may be at risk."));
+    elt.classList.add("headerText", "bottomBorder");
+    box.appendChild(elt);
+
+    elt = doc.createElementNS(XUL_NS, "hbox");
+      elt2 = doc.createElementNS(XUL_NS, "image");
+      elt2.setAttribute("flex", "0");
+      elt2.className = "breachLogo";
+      this.logoElt = elt2;
+      elt.appendChild(elt2);
+
+      elt2 = doc.createElementNS(XUL_NS, "vbox");
+      elt2.setAttribute("align", "start");
+        elt3 = doc.createElementNS(XUL_NS, "description");
+        elt3.className = "headerText";
+        this.breachNameElt = elt3;
+        elt2.appendChild(elt3);
+
+        elt3 = doc.createElementNS(XUL_NS, "description");
+        elt3.className = "redText";
+        elt3.appendChild(doc.createTextNode("Breach Date"));
+        elt2.appendChild(elt3);
+
+        elt3 = doc.createElementNS(XUL_NS, "description");
+        this.breachDateElt = elt3;
+        elt2.appendChild(elt3);
+      elt.appendChild(elt2);
+    box.appendChild(elt);
+
+    elt = doc.createElementNS(XUL_NS, "description");
+    elt.className = "redText";
+    elt.appendChild(doc.createTextNode("Compromised Accounts"));
+    box.appendChild(elt);
```

Comment:
This is a bit boiler-platey... not sure if we want to refactor it out into something more reusable and DRY-er:

```js
function createLabel(label) {
  const el = doc.createElementNS(XUL_NS, "description");
  el.className = "redText";
  el.appendChild(doc.createTextNode(label));
  return el;
}

box.appendChild(createLabel("Compromized Accounts"));
```